docs/source/guide/ml.md

```diff
@@ -111,6 +111,10 @@ Using the project ID and the URL for the machine learning backend, you can also
 
 After you [connect a model to Label Studio as a machine learning backend](#Add-an-ML-backend-to-Label-Studio) and annotate at least one task, you can start training the model. 
 
+### Classical training approach
+
+Classical training approach is when you add annotations in your project and then train your model with this batch of annotations.
```

Comment:
The classical training approach is applicable when you add annotations to your project and then train your model with this batch of annotations.

Reply:
Fixed